### DISCUSSION TITLE
SSE and/or websocket support for endpoints

### DISCUSSION THREAD

**Discussion by FeldrinH** [withastro/roadmap#388] [Proposal]
It would be nice if Astro endpoints supported server sent events or websockets, to allow for components that need to update data live as it changes. Is this something that is on the roadmap for Astro?

**Comment by florian-lefebvre** [comment]
I think SSE works with the Node adapter already. As for websockets, probably better to consolidate the request in 1 place, see https://github.com/withastro/roadmap/discussions/695
